.github/workflows/update_chains_preconfigured.yaml

```
@@ -0,0 +1,62 @@
+name: Update chains preconfigured
+
+on:
+  schedule:
+    - cron: '0 8 1/3 * *'
+  workflow_dispatch:
+  pull_request:
+
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-preconfigured:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout current repository to Master branch
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: 🛠 Set up actual paths
+        uses: ./.github/workflows/setup-path
+
+      - name: ⚙️ Install dependencies
+        run: make init
+
+      - name: 🦾 Update files and data in chains preconfigured
+        run: make update-chains-preconfigured
```

Add step with clear directory:
```suggestion
      - name: ⚙️ Install dependencies
        run: make init
        
      - name: Clear old files
        run: rm -r chains/${{ env.CHAINS_VERSION }}/preConfigured/*

      - name: 🦾 Update files and data in chains preconfigured
        run: make update-chains-preconfigured
```